fbuapp/View Controllers/LogViewController.m

```diff
@@ -0,0 +1,61 @@
+//
+//  LogViewController.m
+//  fbuapp
+//
+//  Created by belchercd on 7/19/19.
+//  Copyright © 2019 eazheng. All rights reserved.
+//
+
+#import "LogViewController.h"
+#import <FBSDKCoreKit/FBSDKCoreKit.h>
+#import <FBSDKLoginKit/FBSDKLoginKit.h>
+#import "ProfileViewController.h"
+#import "Parse/Parse.h"
+
+@interface LogViewController ()
+
+@end
+
+@implementation LogViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    //create facebook login button
+    FBSDKLoginButton *loginButton = [[FBSDKLoginButton alloc] init];
+    loginButton.delegate = self;
+    //contected to a visibe view
+    loginButton.center = self.loginButtonView.center;
+    //ask user for access to information
+    loginButton.permissions = @[@"public_profile"];
+    loginButton.permissions = @[@"email"];
+    //displays the created button
+    [self.view addSubview:loginButton];
+}
+
+
+- (void)loginButton:(FBSDKLoginButton *)loginButton didCompleteWithResult:(nullable FBSDKLoginManagerLoginResult *)result error:(nullable NSError *)error {
+    
+    // take user to profile after successful login
+    if (error) {
+        NSLog(@"%@", error.localizedDescription);
+    }
+    else {
```

Comment:
Should this all be done in AppDelegate? idk

Reply:
For after merging: do we need to check if the user already has an account? If so, it should go to home timeline view; otherwise, go to registration page.

Reply:
Yeah! We'll need to route to tab bar if user is existing and doesn't need signup flow. Great catch Emily!